src/updaters/node/package-lock-json.ts

```diff
@@ -34,6 +34,16 @@ export class PackageLockJson extends DefaultUpdater {
     if (parsed.lockfileVersion === 2 || parsed.lockfileVersion === 3) {
       parsed.packages[''].version = this.version.toString();
     }
+    if (this.versionsMap) {
+      for (const [p, obj] of Object.entries(parsed.packages)) {
```

Comment:
reading this not after midnight, I can probably clean this up a bit. But before I do I would like to get a general go ahead from a maintainer.

Reply:
Feel free to clean this up in a follow up

Reply:
Ok, yeah I don't think it was too bad, but it took me a second in the morning to read that and remember what exactly I had done. If you are good with it on this PR I honestly am good with just leaving it since it really isn't that bad and does appear to fully resolve the issue.